care/users/reset_password_views.py

```
@@ -179,7 +179,15 @@ def post(self, request, *args, **kwargs):
         if user and user.is_active:
             active_user_found = True
             mail_type = MailTypeChoices.reset.value
-            send_password_reset_email(user, mail_type)
+            try:
+                send_password_reset_email(user, mail_type)
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)